【課題提出】其の四 『WEBページの基本的なレイアウト2』安藤

andoazumi opened the pull request:
@tommyKazuhito 
@usukesato 

課題其の四 『WEBページの基本的なレイアウト2』の作成を行いましたので、ご確認よろしくお願いいたします。
制作時間6時間

contents3のところに一番苦戦しました。中央揃えかつ余白固定をどうしたらできるのかわからず「flex 最後だけ中央」「flex justify-content center 余白」などで検索していました。検索結果の中で一番レイアウトに近づける方法で作成いたしました。

contentsのところは全体的に余白の作成、その引き算で時間がかかっております。計算ミスが多かったため、それなりに書き直して時間を消費してしまいました。また命名規則の改善をすべくどの単語を使った方がふさわしいかでも悩み、少し時間がかかりました。

andoazumi committed:
[Add]其の四 『WEBページの基本的なレイアウト2』の作成

【html】
body内の作成

【css】
二行目
:rootの追加

16行目以降　作成

andoazumi commented:
61行目

誤字してる上に消し忘れです。確認不足です。気をつけます。

tommyKazuhito commented on basic-layout/no4/htdocs/css/style.css:
```diff
@@ -14,3 +31,295 @@ body {
+
+.main-visual {
+  background: var(--c-main-visual);
+  height: 100%;
```
@andoazumi 
この100%の指定は不要？

tommyKazuhito commented on basic-layout/no4/htdocs/css/style.css:
```diff
@@ -14,3 +31,295 @@ body {
+
+  .contents-03 {
+    height: 250px;
+    width: calc(33.3% - 20px);
```
@andoazumi 
細かいんですが、33.3%で指定すると合計したときに100より小さくなってしまいます。
なので、100% / 3 で限りなく合計が100になるようにしてあげましょう！

tommyKazuhito commented on basic-layout/no4/htdocs/css/style.css:
```diff
@@ -14,3 +31,295 @@ body {
+
+  .contents-03 {
+    height: 350px;
+    width: calc(25% - 28px);
```
@andoazumi 
サイズがずれちゃってるかな？

![cap](https://user-images.githubusercontent.com/22440274/193486224-3639f419-e23f-49c2-8a58-d514e2a67512.jpg)

tommyKazuhito reviewed (commented):
@andoazumi 
確認しました！
HTMLは大丈夫かなと思います！

数カ所コメントつけましたので、
ご確認を。

あとSPでは、CONTENTS3のテキスト2行になるようにしてみましょう！

andoazumi commented:
@tommyKazuhito 
ご確認とアドバイスいただき、ありがとうございます。
行っていて仕様の方も確認不足なのが出てしまっていると痛感しておりますので、気をつけます。

修正して再提出いたしますので、よろしくお願いいたします。

andoazumi committed:
[Update]課題1_4訂正

【css】
31行目
CONTENTS 3のspの時にだけ効果を発揮してほしいので、97行目へ移しました。

49行目
不要なので削除いたしました。

61行目
不要なので削除いたしました。

232行目
助言いただき、100に近づけるように訂正いたしました。

308行目
数値を変更しました。
28×3÷4で考えました。

andoazumi commented:
@tommyKazuhito
@usukesato

お疲れ様です。
課題1_4を再提出しましたので、お手隙の際にご確認の程よろしくお願いいたします。

tommyKazuhito reviewed (approved):
@andoazumi 
LGTM!
